Dockerfile

```diff
@@ -66,7 +66,7 @@ RUN --mount=target=.,from=src,src=/src/qemu,rw --mount=target=./install-scripts,
 ARG BINARY_PREFIX
 RUN cd /usr/bin; [ -z "$BINARY_PREFIX" ] || for f in $(ls qemu-*); do ln -s $f $BINARY_PREFIX$f; done
 
-FROM --platform=$BUILDPLATFORM golang:1.16-alpine AS binfmt
+FROM --platform=$BUILDPLATFORM golang:1.17-alpine AS binfmt
```

Comment:
Could you move this into build arg like in https://github.com/docker/buildx/blob/master/Dockerfile#L3 so we only need to update one place in the future.